Article/Article.js

```diff
@@ -3,24 +3,26 @@
 class Article {
   constructor(domElement) {
     // assign this.domElement to the passed in domElement
-    this.domElement;
+    this.domElement = domElement;
     // create a reference to the ".expandButton" class. 
-    this.expandButton;
+    this.expandButton = this.domElement.querySelector('.expandButton')
     // Using your expandButton reference, update the text on your expandButton to say "expand"
-    
+    this.expandButton.innerText = 'Expand'
     // Set a click handler on the expandButton reference, calling the expandArticle method.
-    
+    this.expandButton.addEventListener('click', () => {
+      this.expandArticle(event);
+    })
   }
 
   expandArticle() {
     // Using our reference to the domElement, toggle a class to expand or hide the article.
-
+    this.domElement.classList.toggle('article-open')
   }
 }
 
 // START HERE: Select all classes named ".article" and assign that value to the articles variable
-let articles;
-
+let articles = document.querySelectorAll('.article')
 // Use .map() to iterate over the articles array and create a new instance of Article by passing in each article as a parameter to the constructor.
-articles = Array.from(articles);
-
+articles = Array.from(articles).map(domElement => {
```

Comment:
technically, if you use map, you should use `return new Article(domElement)`. Otherwise you can just use forEach